routes/web.php

```diff
@@ -60,21 +60,20 @@
 /*
 *   Controller Dashboard
 */
-Route::prefix('dashboard')->middleware('auth')->group(function() {
+Route::prefix('dashboard')->middleware('auth')->group(function () {
     Route::get('/', 'ControllerDash@dash');
 
-    Route::prefix('controllers')->group(function() {
-      
+    Route::prefix('controllers')->group(function () {
         Route::get('/mentoravi', 'TrainingController@showMentAvail');
         Route::post('/mentoravi', 'TrainingController@saveSession');
         Route::get('/trainingreq', 'TrainingController@showRequests');
         Route::delete('/trainingreq/{id}/delete', 'TrainingController@cancelSession');
         Route::get('/sessions_today', 'TrainingController@showsessions_today');
 
```

Comment:
Will you please remove all of these extra lines? I'm not sure why they are in here... (not your fault, but we might as well fix it if we are fixing formatting)